### PR TITLE
[CCTRI-1136]  Implement /health endpoint

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**

### CODE REVIEW

File: api/client.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        self.data = {
+            **current_app.config['REQUEST_DATA'],
+            'key': api_key
+        }
+
+    def get_c1fapp_response(self, observable):
+        self.data.update({'request': observable})
+
+        response = requests.post(
+            self.api_url, headers=self.headers, data=json.dumps(self.data)
```
Comment (gdavoian) [inline api/client.py:24]: You can simply use `json=self.data` and the library will serialize your data into JSON (+ add the proper Content-Type header) for you.

File: api/client.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        )
+
+        if not response.ok:
+            raise UnexpectedC1fAppError(response)
+
+        return response.json
```
Comment (gdavoian) [inline api/client.py:30]: You're returning a method instead of calling it :)

Comment (AnastasiiaRozlyvan) [reply api/client.py]: Fixed, thanks!

File: api/client.py (hunk 1 above)
Comment (AnastasiiaRozlyvan) [reply api/client.py]: Yes, it will be much better. Corrected.

**Pull request merged**